src/mailing_info.cpp

```diff
@@ -55,14 +55,21 @@ void replace_all_irefs(std::vector<lwg::issue> const & issues, std::string & s)
 } // close unnamed namespace
 
 auto lwg::make_html_anchor(lwg::issue const & iss) -> std::string {
-   auto temp = std::to_string(iss.num);
+   auto num = std::to_string(iss.num);
+   auto title = iss.title;
+   for (auto p = title.find('<'); p != title.npos; p = title.find('<', p))
+      title.erase(p, title.find('>', p) + 1 - p);
+   for (auto p = title.find('"'); p != title.npos; p = title.find('"', p))
+      title.replace(p, 1, "&quot;");
```

Comment:
Ditto "we should replace ampersands before adding them".

Reply:
The code above needs to replace `"` in the titles because you can't have `"` inside `title="..."` . It strips out everything between `<` and `>` because issue titles _are_ written in XML, so have elements like `<code>` and `<i>` which we don't want in title attributes.

But it doesn't actually need to remove ampersands.

It would be more correct for XML, but we aren't actually using a real XML parser. For the HTML output  unescaped ampersands in attribute text are OK (HTML5 allows them as long as the characters following them can't be interpreted as an entity ... although we should replace the HTML 4 DTD at the top of the HTML output so that we can actually use HTML5).

Reply:
And actually the text being processed here is already supposed to be valid HTML or XML, so any ampersands in it are actually entity references. Replacing those ampersands will break them.

e.g. for issue 3099 the attribute will now be:

    title="is_assignable&amp;lt;Incomplete&amp;amp;, Incomplete&amp;amp;&amp;gt;"

This is garbage. Replacing the ampersands there in 582516611ea3e702234292e373d3f8084f27db72 was wrong.

For the paper titles coming from `paper_titles.txt` we can safely replace ampersands. It's not needed for any current paper titles, but that could change.

Reply:
Escaping ampersands in the paper titles will produce some garbage too, e.g. for P2795R0 we would get:

    title="Correct and incorrect code, and &quot;erroneous behaviour&quot;"

But that's a problem in the original paper title.

For P1272R4 we'll get:

    title="Byteswapping for fun&&nuf"

and this is valid in HTML5, neither of those ampersands is ambiguous. Escaping them is OK too.

Reply:
I'm convinced. (I'd resolve this comment if I were a maintainer or author - but I can't.)

Reply:
Done, but I'll revert it next time I push.